pages/details/_title.vue

```diff
@@ -3,46 +3,57 @@
     <v-card elevation="2">
       <v-img
         class="white--text align-end"
-        height="200px"
-        :src="singleData.urlToImage"
+        :src="product.urlToImage"
       >
-        <v-card-title>{{ singleData.title }}</v-card-title>
+        <v-card-title>{{ product.title }}</v-card-title>
       </v-img>
 
       <v-card-subtitle class="pb-0">
-        {{ singleData.description }}
+        {{ product.description }}
       </v-card-subtitle>
 
-      <v-card-text class="text--primary">{{ singleData.author }} </v-card-text>
+      <v-card-text class="text--primary">{{ product.author }} </v-card-text>
       <h5 class="date-v">Published date : 20th January 2023</h5>
```

Comment:
please make the date related changes here as well.

Reply:
Resolved